cli/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "subsembly-cli",
+  "version": "1.0.0",
+  "description": "CLI tool for Subsembly",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "ts-node src/index.ts"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/oclif/example-single-ts.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/oclif/example-single-ts/issues"
+  },
+  "homepage": "https://github.com/oclif/example-single-ts#readme",
```

Comment:
You can add `keywords`, `author`, `license -> apache`, the correct link for bugs -> under `subsembly-github/issues` and for homepage we could have the github link -> readme